pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java

```diff
@@ -838,88 +838,6 @@ public void testActiveAndInActiveConsumerEntryCacheBehavior() throws Exception {
         log.info("-- Exiting {} test --", methodName);
     }
 
-    @Test
-    public void testDeactivatingBacklogConsumer() throws Exception {
```

Comment:
should we keep this test as broker still have backlog cursor check logic.

Reply:
Readded the test